### DISCUSSION TITLE
Do I need to keep mjs extension ?

### DISCUSSION THREAD

**Discussion by dante01yoon** [chakra-ui/panda#904] [Q&A] [answered]
Hi, `npx panda init` gives me all mjs files below.

I wonder why panda's scaffold files are all mjs, not a js.

Furthermore, can I remove scaffold files using panda.css?
![스크린샷 2023-07-02 오전 12 14 02](https://github.com/chakra-ui/panda/assets/6510430/f3d988e5-86f9-4e24-86b8-227e28c9d527)

**Comment by anubra266** [comment, marked as answer]
You can configure the extension if you prefer to use js: https://panda-css.com/docs/references/config#outextension
I don't understand what you mean by removing scaffold files.

**Reply by dante01yoon** [reply]
thank you!
